core-java-modules/core-java-jvm/src/test/java/com/baeldung/bytecode/ViewBytecodeUnitTest.java

```diff
@@ -43,7 +43,7 @@ public void whenUsingJavassist_thenReadBytecode() throws NotFoundException {
         ClassFile cf = cp.get("java.lang.Object").getClassFile();
         
         assertEquals(cf.getName(), "java.lang.Object");
-        assertEquals(cf.getMethods().size(), 14);
+        assertEquals(cf.getMethods().size(), 12);
```

Comment:
is this needed? do we need any article change for this?

Reply:
Yes, it needed with the new Java version.
Yes, the article is: https://www.baeldung.com/java-class-view-bytecode